thirdeye/thirdeye-spi/src/main/java/org/apache/pinot/thirdeye/datalayer/dto/AlertSnapshotDTO.java

```diff
@@ -80,48 +75,6 @@ public void setSnapshot(Multimap<String, AnomalyNotifiedStatus> snapshot) {
     }
   }
 
-  public void updateSnapshot(DateTime alertTime, List<MergedAnomalyResultDTO> alertedAnomalies) {
```

Comment:
Why move this?

Reply:
primarily to disentangle the code. This class used to depend on a Resource! And this method is used in a single place.